VerifyBamID/2.0.1--h32f71e1_0/CheckContamination.nf

```diff
@@ -0,0 +1,64 @@
+process CheckContamination {
+    tag {"VerifyBamID CheckContamination ${sample_id}.${int_tag}"}
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0'
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0_CheckContamination'
+    container = 'quay.io/biocontainers/verifybamid2:2.0.1--h32f71e1_0'
+    shell = ['/bin/bash', '-eo', 'pipefail']
+
+    input:
+        tuple (
+            sample_id, 
+            path(bam), 
+            path(bai))
+
+    output:
+        tuple (
+            sample_id, 
+            path("${output_prefix}.selfSM"), 
+            stdout)
```

Comment:
example output:

```
[U175754PMgiab12891, 
[/path/to/nf-testrun/work/3c/dccac4e3aed877da6c1db3656644b0/U175754PMgiab12891.WES.contamination.selfSM], [0.00015529733333333333]
]
```

Reply:
stdout is coming from the python script part?

Reply:
Yes.

Reply:
- [ ] one line